src/Castle.Core/DynamicProxy/Internal/AttributeUtil.cs

```diff
@@ -234,6 +234,11 @@ private static bool ShouldSkipAttributeReplication(Type attribute)
 				return true;
 			}
 
+			if (SpecialCaseAttributeThatShouldBeReplicated(attribute))
+			{
+				return false;
+			}
```

Comment:
... or should this be placed a little further up in so it would take preference over `SpecialCaseAttributeThatShouldNotBeReplicated`?